egs/tedlium/s5/local/confidence_calibration.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+. cmd.sh
+. path.sh
+
+# Global options,
+graph=exp/tri3/graph
+arpa_gz=db/cantab-TEDLIUM/cantab-TEDLIUM-pruned.lm3.gz
+lmwt=13
+
+# Dev-set options,
+dev_data=data/dev
+dev_latdir=exp/tri3_mmi_b0.1/decode_dev_it4
+
+# Eval-set options,
+eval_data=data/test
+eval_latdir=exp/tri3_mmi_b0.1/decode_test_it4
+
+. utils/parse_options.sh
+set -euxo pipefail
+
+# Derived options,
+dev_caldir=$dev_latdir/confidence_$lmwt
+eval_caldir=$eval_latdir/confidence_$lmwt
+
+###### Train the calibration,
+steps/conf_train_calibration.sh --cmd "$decode_cmd" --lmwt $lmwt \
```

Comment:
There should probably be a subdirectory here, e.g. steps/conf/.
I am moving towards using subdirectories, as steps/ is getting over-full.
Within sub-directories, it's OK to put in steps/, scripts that would traditionally have gone in utils/ (i.e. scripts that are utility-style scripts that don't take the "$cmd" option).